src/scripts/clipperUI/componentBase.ts

```diff
@@ -75,12 +76,20 @@ export abstract class ComponentBase<TState, TProps> {
 	 * Example use:
 	 *      <a id="myCoolButton" {...this.enableInvoke(this.myButtonHandler, 0)}>Click Me</a>
 	 */
-	public enableInvoke({callback = undefined, tabIndex = 0, args = undefined, idOverride = undefined, setNameForArrowKeyNav = undefined}: EnableInvokeParams) {
+	public enableInvoke({callback = undefined, tabIndex = 0, args = undefined, idOverride = undefined, verticalArrowKeyNavSet = undefined, horizontalArrowKeyNavSet = undefined}: EnableInvokeParams) {
```

Comment:
I think the vertical/horizontal can be parameterized in a better way. Right now, it's possible to set defined values for both params, which will result in unclear behaviour